src/web/src/app/app.component.html

```diff
@@ -36,26 +36,36 @@ <h1 class="branding__text" *ngIf="!mobileQuery.matches">SvetIT</h1>
     <span class="spacer"></span>
 
     <div class="app-toolbar__lang-select">
-      <button mat-icon-button (click)=select.open()>
+      <mat-select #select class="langSelect" [(ngModel)]="current_lang_" (ngModelChange)="set_language()">
+		<mat-option *ngFor="let lang of languages" [value]="lang">
+		  <span class="{{lang.icon}}"></span> {{ lang.label }}
+		</mat-option>
+	  </mat-select>
+	  <button class="langSelectButton" mat-icon-button (click)=select.open()>
         <mat-icon *ngIf="current_lang_ === undefined">language</mat-icon>
-        <span *ngIf="current_lang_ !== undefined" [class]="current_lang_.icon"></span>
-        <mat-select #select class="langSelect" [(ngModel)]="current_lang_" (ngModelChange)="set_language()">
-          <mat-option *ngFor="let lang of languages" [value]="lang">
-            <span class="{{lang.icon}}"></span> {{ lang.label }}
-          </mat-option>
-        </mat-select>
+        <span *ngIf="current_lang_ !== undefined" [class]="current_lang_.icon"></span>    
```

Comment:
Проверь пожалуйста чтобы везде в новом коде были табы вместо пробелов и чтобы не было пробелов в конце строки.
```suggestion
	 <span *ngIf="current_lang_ !== undefined" [class]="current_lang_.icon"></span>
```

Можно попробовать как то автоматически если если идее или какой нибудь вот такой командой:
```bash
git -c core.whitespace=indent-with-non-tab diff --ws-error-highlight=all develop
```

Хотя такая команда почему то не всегда показывает. Например в этом файле в строке 41 не сработала. ((

Reply:
Готово.